rsv-core/lib/python/rsv/rsv_control.py

```diff
@@ -151,9 +151,9 @@ def process_options(arguments=None):
             parser.error("You must provide a list of metrics to run or else pass the " +
                          "--all-enabled flag to run all enabled metrics")
 
-    if options.ce_type and options.ce_type not in ('gram', 'condor-ce', 'htcondor-ce'):
+    if options.ce_type and options.ce_type not in ('gram', 'condor-ce', 'htcondor-ce', 'cream', 'nordugrid'):
         parser.error("Invalid value for --ce-type. "
-                     "Valid values are 'gram' for Globus GRAM, and 'htcondor-ce' (or 'condor-ce') for HTCondor-CE")
+                     "Valid values are 'gram' for Globus GRAM, 'htcondor-ce' (or 'condor-ce') for HTCondor-CE, 'cream' for CREAM-CE and nordugrid for nordugrid")
```

Comment:
Should be ...`'nordugrid' for Nordugrid` (missing the quotes and the capitalization)